### PR TITLE
Make threshold for Sliders slidesMoved adjustable via moveThreshold prop

### PR DESCRIPTION
**Pull request by maps82**
I my project, the slides are quite big. To trigger a scroll to the next slide you have to drag over quite a long distance. Too long for my taste :)

So I created this PR that makes the hard coded threshold used in `slidesMoved` adjustable via a `moveThreshold` prop. I hope you like it.

### CODE REVIEW

File: src/Slider/Slider.jsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ const Slider = class Slider extends React.Component {
     isPlaying: PropTypes.bool.isRequired,
     lockOnWindowScroll: PropTypes.bool.isRequired,
     masterSpinnerFinished: PropTypes.bool.isRequired,
+    moveThreshold: PropTypes.number.isRequired,
```
Comment (tim-steele) [inline src/Slider/Slider.jsx]: If we are going to have a defaultProp for it and it says in the table it is not required, shouldn't this not have `isRequired`?

**Comment by tim-steele** [comment]
@maps82 One little change but I think it looks good.

File: src/Slider/Slider.jsx (hunk 1 above)
Comment (maps82) [reply src/Slider/Slider.jsx]: I removed `isRequired`.

**Comment by tim-steele** [comment]
@maps82 I published 1.14.0 with the change. Thanks!

**Pull request merged**